### PR TITLE
feat(cli): If execute fails, output command line with all vars as --set args

### LINKED ISSUE #53: If execution of recipe fails, output jalapeno command line with all vars so it's easy to re-run
The point here was that if one is developing a recipe, one might run a broken version from time to time. It would be annoying to have to enter all the vars in the TUI every time, so probably for testing one would use `jalapeno execute --set ...` instead. But wouldn't it be nice if, when you used the TUI _and_ the recipe was broken somehow, jalapeno printed the equivalent command line using `--set`? That's the point here.

### PR DESCRIPTION
**Pull request by ilkka**
This is a convenience feature for recipe authors: if recipe execution fails (e.g. due to a bug in a template), and some possibly complex variable values were entered, it would be nice to not have to enter them again in order to verify a fix. This change outputs a command line with `--set` args for each variable whether or not it was originally passed on the command line or set through the TUI.

Closes #53.

### CODE REVIEW

**Comment by ilkka** [comment]
Oh poo, the quoting breaks the test.

File: internal/cli/execute.go [hunk 1]
```diff
@@ -177,3 +179,32 @@ func runExecute(cmd *cobra.Command, opts executeOptions) error {
...
+		if idx <= 1 || strings.HasPrefix(arg, "-") {
+			commandline.WriteString(arg)
+		} else {
+			commandline.WriteString(fmt.Sprintf("\"%s\"", arg))
+		}
+		commandline.WriteString(" ")
+	}
+
+	for key, value := range values {
+		commandline.WriteString(fmt.Sprintf(" --set \"%s=%s\"", key, value))
```
Comment (majori) [inline internal/cli/execute.go:206]: For more complex variable types, we need to make sure that the value is serialized correctly. For example for table variables we need to serialize it to CSV format. Currently it outputs `--set "project_name=[map[1:a 2:b]]"`

**Review by majori — CHANGES_REQUESTED** [review]

File: internal/cli/execute.go [hunk 2]
```diff
@@ -146,13 +146,15 @@ func runExecute(cmd *cobra.Command, opts executeOptions) error {
...
-		return err
+		retryMessage := makeRetryMessage(values)
+		return fmt.Errorf("%w\n\n%s", err, retryMessage)
 	}
 
 	// Check for conflicts
 	for _, s := range existingSauces {
 		if conflicts := s.Conflicts(sauce); conflicts != nil {
-			return fmt.Errorf("conflict in recipe '%s': file '%s' was already created by recipe '%s'", re.Name, conflicts[0].Path, s.Recipe.Name)
+			retryMessage := makeRetryMessage(values)
```
Comment (majori) [inline internal/cli/execute.go:156]: Could we also add this to `upgrade` command?

File: internal/cli/execute.go (hunk 1 above)
Comment (ilkka) [reply internal/cli/execute.go:206]: Heck, I thought my godog test tested that, but I guess it doesn't.

File: internal/cli/execute.go (hunk 2 above)
Comment (ilkka) [reply internal/cli/execute.go:156]: Sure! Should be no problem now that the message generator function doesn't depend on anything.

File: internal/cli/execute.go (hunk 1 above)
Comment (ilkka) [reply internal/cli/execute.go]: I removed the test for now, I couldn't think of a way to test this in godog :(

File: internal/cli/execute.go (hunk 2 above)
Comment (ilkka) [reply internal/cli/execute.go:156]: It's added to a bunch of places in upgrade now too.

**Pull request merged**